pkg/dns/dns_proxy_handler.go

```diff
@@ -109,7 +110,9 @@ func getUpdateCacheFunc(log logr.Logger, cache *DNSCache) func(lookupTime time.T
 	return func(lookupTime time.Time, response *dnsgo.Msg) {
 		if response.Response && response.Rcode == dnsgo.RcodeSuccess {
 			scopedLog := log.WithValues(reqIdLogField, response.Id)
-			if err := cache.Update(lookupTime, response); err != nil {
+			qname := strings.ToLower(response.Question[0].Name)
+			log.Info("DEBUG dnsproxyhandler function getUpdateCacheFunc updating DNS cache", "queried name", qname, "dns response", response)
```

Comment:
For debug logging you can raise the verbosity like this: `log.V(4).Info("...")`. Default info level is V(0), V(4) is mostly used for debugging. See: https://pkg.go.dev/github.com/go-logr/logr#hdr-Verbosity

Reply:
Done!